mobile/function/map_page_function.js

```diff
@@ -0,0 +1,159 @@
+import { Text, View, Image, Pressable } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import { styles } from "../styles/basic_styles";
+import MapView, { Geojson, Marker } from "react-native-maps";
+import { map } from "../styles/map_page_styles";
+import { useCallback, useEffect, useState } from "react";
+import { StatusBar } from "expo-status-bar";
+import { FontAwesome5 } from "@expo/vector-icons";
+import { IP } from "@env";
+
+/**
+ * shows the map with the route and all the markers
+ * @returns a map with geojson
+ */
+const GetMapPage = (routeId) => {
+    const nav = useNavigation();
+    const [data, setData] = useState([]);
+    const [isLoading, setLoading] = useState(true);
+
+    //fetch the data and catch errors
+    useEffect(() => {
+        fetchData()
+            .catch((error) => console.error(error))
+            .finally(() => setLoading(false));
+    }, [fetchData]);
+
+    //Fetching the data from the api.
+    const fetchData = useCallback(async () => {
+        const response = await fetch(
+            "http://" + IP + ":3000/api/routes/route/" + routeId
```

Comment:
waarom is het in "http://" + IP + ":3000/api/routes/route/" + routeId. En niet `http://" + IP + ":3000/api/routes/route/${routeId}`?

Reply:
Zie mijn bericht